lib/swaps/Swaps.ts

```diff
@@ -96,6 +101,16 @@ class Swaps extends EventEmitter {
     return { takerAmount, makerAmount };
   }
 
+  public init = async () => {
+    // Load Swaps from data base
+    const promises: PromiseLike<any> = this.repository.getSwaps();
+    const result = await Promise.resolve(promises);
```

Comment:
Can we collapse these two lines into one by just calling `const swapDeals = await this.repository.getSwaps();`?  There's only a single promise here so we don't really need a `promises` variable.